README.md

```diff
@@ -112,17 +112,6 @@ end
 
 ## Development
 
-Before installing dependencies, make sure your environment variables are setup.
-Our CI uses something like this:
-`SOLIDUS_BRANCH=v2.10 DB=postgresql RAILS_VERSION="~> 5.2.0"`.
-
-**NOTE: These environment variables need to be set when installing dependencies
-and running specs, so set them before each command like this:**
-
-```sh
-SOLIDUS_BRANCH=v2.10 DB=postgresql RAILS_VERSION="~> 5.2.0" bin/setup
-```
```

Comment:
I think it's worth mentioning that these environment variables need to always be set/exported when running commands otherwise the fallbacks will take effect, which may cause confusion and unexpected behaviour. We've all wasted enough time with this so I think at least part of this warning is worth preserving.

Reply:
We've resolved this issue in another PR. Now, the default environment variables make exporting other variables unnecessary.